lib/zephyrus/resources/output.rb

```diff
@@ -0,0 +1,14 @@
+module Zephyrus
+  class Output < Base
+    field :id
+    field :created_at
+    field :updated_at
+    field :url
+    field :key
+    field :uid
+    field :state
+    field :input_id
+    field :type
+    field :properties
+  end
+end
```

Comment:
This is good but I think we will need a resource for each output type ( RoutingOutput & TranscodingOutput ). You can look at how this works with Advertising::DoubleClickAdvertisement resource for example